app/Controllers/Http/UsersController.ts

```diff
@@ -1,16 +1,57 @@
-import { HttpContextContract } from '@ioc:Adonis/Core/HttpContext'
+import User from 'App/Models/User';
+import { schema, rules } from '@ioc:Adonis/Core/Validator';
 import Database from '@ioc:Adonis/Lucid/Database'
 export default class UserController {
-    public async index(ctx: HttpContextContract) {
-        const postId = await Database
-            .table('users')
-            .insert({
-                email: 'Adonis 101',
-                password: 'Let\'s learn AdonisJS',
-                created_at:'12/05/2022',
-                updated_at:'12/05/2022'
+    public async store({ request, response }) {
+        const newUserSchema = schema.create({
+            mobile: schema.string([
+                rules.minLength(10),
+                rules.maxLength(10),
+                rules.unique({ table: 'users', column: 'mobile' })
+            ])
+        })
+        try {
+            const payload = await request.validate({
+                schema: newUserSchema
             })
-            .returning('id')
-            console.log(postId)
+            await User.create(payload);
+            response.send({ message: 'User created', status: 'success', statusCode: 200 })
+
+        } catch (error) {
+            response.badRequest(error.messages)
+        }
+    }
+    public async login({ auth, request, response }) {
+        console.log(request.body().mobile)
+        // const data1 = await Database.from('api_tokens').where({'expires_at' < raw(`Date.now()`)});
+        // console.log(data1)
+        try {
+            const mobile = request.body().mobile
```

Comment:
Validate if the mobile provided in request body is proper 10 digit number or not.

Reply:
Yes already added minlength and maxlength validation

Reply:
The validation is done for Register API. We need to do the same for Login API.